src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRow.tsx

```diff
@@ -53,8 +53,7 @@ export const TransactionRow = ({
 				<div className="inline-block align-middle">
 					<Link
 						data-testid="TransactionRow__ID"
-						to=""
-						onClick={onClick}
+						to={{ pathname: transaction.explorerLink() }}
```

Comment:
```suggestion
						to={transaction.explorerLink()}
```

should be enough ☝️

Reply:
Right :+1: Needs also to stop propagate the onClick event for the row as it still throws the error